src/NzbDrone.Core/Notifications/Gotify/GotifyService.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using FluentValidation.Results;
+using NLog;
+using RestSharp;
+using NzbDrone.Core.Rest;
+
+namespace NzbDrone.Core.Notifications.Gotify
+{
+    public interface IGotifyProxy
+    {
+        void SendNotification(string title, string message, GotifySettings settings);
+        ValidationFailure Test(GotifySettings settings);
+    }
+
+    public class GotifyProxy : IGotifyProxy
+    {
+        private readonly Logger _logger;
+
+        public GotifyProxy(Logger logger)
+        {
+            _logger = logger;
+        }
+
+        public void SendNotification(string title, string message, GotifySettings settings)
+        {
+            var URL = settings.GotifyServer + "/message?token=" + settings.AppToken;
```

Comment:
Variables should be camel case, `url` not `URL`. Use `string.Format` better yet string interpolation.

Reply:
Changed this to use string interpolation to build the url.